invenio_app_ils/ui/src/invenio_app_ils/pages/frontsite/Documents/DocumentsDetails/DocumentsDetailsContainer.js

```diff
@@ -40,33 +41,38 @@ export default class DocumentsDetailsContainer extends Component {
     this.setState({ searchQuery: event.target.value });
   }
 
+  breadcrumbs = () => [
+    { to: FrontSiteRoutes.home, label: 'Home' },
+    { to: FrontSiteRoutes.documentsList, label: 'Search' },
+  ];
+
   _renderElement(isLoading, error, data) {
     return (
-      <Container>
-        <SearchBar
-          currentQueryString={this.state.searchQuery}
-          onInputChange={this.onSearchInputChange}
-          executeSearch={this.onSearchClick}
-          placeholder={'Search for books...'}
-        />
-        <br />
-        <Loader isLoading={isLoading}>
-          <Error error={error}>
-            <Breadcrumb>
-              <Breadcrumb.Section link>Home</Breadcrumb.Section>
-              <Breadcrumb.Divider icon="right chevron" />
-              <Breadcrumb.Section link>Search</Breadcrumb.Section>
-              <Breadcrumb.Divider icon="right arrow" />
-              <Breadcrumb.Section active>
-                {data.metadata ? data.metadata.title : null}
-              </Breadcrumb.Section>
-            </Breadcrumb>
-            <br />
-            <DocumentPanel />
-            <DocumentMetadata />
-          </Error>
-        </Loader>
-      </Container>
+      <>
+        <Container fluid className="document-details-search-container">
+          <Container>
+            <SearchBar
+              currentQueryString={this.state.searchQuery}
+              onInputChange={this.onSearchInputChange}
+              executeSearch={this.onSearchClick}
+              placeholder={'Search for books...'}
+              className="document-details-search-bar"
+            />
+          </Container>
+        </Container>
+        <Container className={'document-details-container'}>
+          <Loader isLoading={isLoading}>
+            <Error error={error}>
+              <Breadcrumbs
+                elements={this.breadcrumbs()}
+                currentElement={data.metadata ? data.metadata.title : null}
+              />
+              <DocumentPanel />
+              <DocumentMetadata />
+            </Error>
+          </Loader>
+        </Container>
+      </>
```

Comment:
The empty `<></>` is not needed you have the container as root element.

Reply:
actually, when you have 
```
<Container>
</Container>
<Container>
</Container>
```
you need a wrapper `<></>`